### PR TITLE
Enoch/physx windows

### PR DESCRIPTION
**Pull request by enochtsang**

### CODE REVIEW

**Comment by ammunoz** [comment]
Please make sure to include the SDL_Image dll dependencies (zlib, libpng) into the zip file before merging this in.

**Comment by enochtsang** [comment]
We can put that in a separate PR.

File: README.md [hunk 1]
```diff
@@ -11,4 +11,59 @@
...
+│   ├── SDL2.dll
+│   ├── SDL2_image.dll
+│   ├── SDL2_mixer.dll
+│   ├── assimp-vc140-mt.dll
+│   ├── glew32.dll
+│   ├── libjpeg-9.dll
+│   ├── libpng16-16.dll
+│   ├── libtiff-5.dll
+│   ├── libwebp-7.dll
+│   └── zlib1.dll
```
[the remark this answers is not in the document]
Comment (enochtsang) [reply README.md]: No, I need to update this. I want to be detailed so the guys in vancouver can fix it if they're having troubles compiling.

**Comment by enochtsang** [comment]
Replaced by #32

**Pull request closed without merging**